concrete-storage/package-lock.json

```diff
@@ -0,0 +1,6041 @@
+{
```

Comment:
Probably we want to delete this file, and add a `.npmrc` file here with `package-lock = false` following what we did in top level.